pkg/alert/ctl_alert.go

```diff
@@ -89,27 +89,27 @@ func (ctl *HelmValuesFromCobraFlags) SetArgs(args map[string]interface{}) {
-	cmd.Flags().StringVar(&ctl.flagTree.Version, "version", ctl.flagTree.Version, "Version of Alert")
+	cmd.Flags().StringVar(&ctl.flagTree.Version, "version", "5.3.0", "Version of Alert")
 	if master {
 		cobra.MarkFlagRequired(cmd.Flags(), "version")
 	}
 
-	cmd.Flags().StringVar(&ctl.flagTree.StandAlone, "standalone", ctl.flagTree.StandAlone, "If true, Alert runs in standalone mode [true|false]")
+	cmd.Flags().StringVar(&ctl.flagTree.StandAlone, "standalone", "true", "If true, Alert runs in standalone mode [true|false]")
 	if master {
 		cmd.Flags().StringVar(&ctl.flagTree.ExposeService, "expose-ui", util.NONE, "Service type to expose Alert's user interface [NODEPORT|LOADBALANCER|OPENSHIFT|NONE]")
 	} else {
 		cmd.Flags().StringVar(&ctl.flagTree.ExposeService, "expose-ui", ctl.flagTree.ExposeService, "Service type to expose Alert's user interface [NODEPORT|LOADBALANCER|OPENSHIFT|NONE]")
 	}
 	cmd.Flags().StringVar(&ctl.flagTree.EncryptionPassword, "encryption-password", ctl.flagTree.EncryptionPassword, "Encryption Password for Alert")
 	cmd.Flags().StringVar(&ctl.flagTree.EncryptionGlobalSalt, "encryption-global-salt", ctl.flagTree.EncryptionGlobalSalt, "Encryption Global Salt for Alert")
-	cmd.Flags().StringVar(&ctl.flagTree.PersistentStorage, "persistent-storage", ctl.flagTree.PersistentStorage, "If true, Alert has persistent storage [true|false]")
+	cmd.Flags().StringVar(&ctl.flagTree.PersistentStorage, "persistent-storage", "false", "If true, Alert has persistent storage [true|false]")
 	cmd.Flags().StringSliceVar(&ctl.flagTree.Environs, "environs", ctl.flagTree.Environs, "Environment variables of Alert")
 	cmd.Flags().StringVar(&ctl.flagTree.PVCName, "pvc-name", ctl.flagTree.PVCName, "Name of the persistent volume claim")
 	cmd.Flags().StringVar(&ctl.flagTree.PVCStorageClass, "pvc-storage-class", ctl.flagTree.PVCStorageClass, "Storage class for the persistent volume claim")
-	cmd.Flags().StringVar(&ctl.flagTree.PVCSize, "pvc-size", ctl.flagTree.PVCSize, "Memory allocation of the persistent volume claim")
-	cmd.Flags().StringVar(&ctl.flagTree.AlertMemory, "alert-memory", ctl.flagTree.AlertMemory, "Memory allocation of Alert")
-	cmd.Flags().StringVar(&ctl.flagTree.CfsslMemory, "cfssl-memory", ctl.flagTree.CfsslMemory, "Memory allocation of CFSSL")
-	cmd.Flags().StringVar(&ctl.flagTree.Registry, "registry", ctl.flagTree.Registry, "Name of the registry to use for images e.g. docker.io/blackducksoftware")
+	cmd.Flags().StringVar(&ctl.flagTree.PVCSize, "pvc-size", "5G", "Memory allocation of the persistent volume claim")
+	cmd.Flags().StringVar(&ctl.flagTree.AlertMemory, "alert-memory", "2560Mi", "Memory allocation of Alert")
+	cmd.Flags().StringVar(&ctl.flagTree.CfsslMemory, "cfssl-memory", "640Mi", "Memory allocation of CFSSL")
+	cmd.Flags().StringVar(&ctl.flagTree.Registry, "registry", "docker.io/blackducksoftware", "Name of the registry to use for images")
```

Comment:
why we need to set it? by default, it is "docker.io/blackducksoftware"?

Reply:
I put it here so that the customer will see it when they do "--help"

Reply:
ok.. not sure whether we need to do the same change in Black Duck?

Reply:
I can put all the default values into Black Duck. I'll make another PR soon